core/src/tangram.cpp

```diff
@@ -457,6 +457,8 @@ void Map::render() {
 
     impl->labels.drawDebug(impl->renderState, impl->view);
 
+    impl->selectionBuffer.drawDebug(impl->renderState, glm::vec2(impl->view.getWidth(), impl->view.getHeight()));
```

Comment:
We can probably leave that commit and replace the toggle, I prefer how the framebuffer is being visualized here.